src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/service/impl/MessageProcessorServiceImpl.java

```diff
@@ -39,30 +39,30 @@
 
     @Override
     public void processMessage(ChipsRestInterfacesSend message) throws ServiceException {
-        Map<String, Object> logMap = new HashMap<>();
-        logMap.put("Message", message.getData());
+        var messageId = message.getMessageId();
         try {
             chipsRestClient.sendToChips(message);
-        } catch (HttpClientErrorException hcee) {
-            logMap.put("HTTP Status Code", hcee.getStatusCode());
-            handleFailedMessage(message, hcee, logMap);
+        } catch (HttpStatusCodeException hsce) {
+            Map<String, Object> logMap = new HashMap<>();
+            logMap.put("HTTP Status Code", hsce.getStatusCode());
+            logger.error(String.format(SEND_FAILURE_MESSAGE, messageId), hsce, logMap);
```

Comment:
I was speaking to @sroberts2 about logging out of a message, and we decided for the time being that we need the message logged as if the message gets stuck in error mode, we need to know what the message was to fix it.

Reply:
oh

Reply:
We need to leave the error message in place for the moment otherwise we will be unable to support the application in the live environment. We need to think about sanitising the data for sensitive information in future iterations but we will be OK for the moment.